scout/server/app.py

```diff
@@ -35,6 +35,40 @@
 )
 
 LOG = logging.getLogger(__name__)
+USERS_LOGGER_PATH_PARAM = "USERS_ACTIVITY_LOG_PATH"
+SUB_URL_IGNORE_LIST = [
+    "static",
+    "ideograms",
+    "custom_images",
+    "autozygous_images",
+    "coverage_images",
+    "upd_regions_images",
+    "favicon",
+    "GET /",
+    "Closing",
+    "Enable",
+    "enabled",
+    "Collecting IGV tracks",
```

Comment:
I dunno, some of these aren't really sub URLs. They are, shall we say, filter ignore triggers?

Reply:
Ah we can rename it, no problem!